lib/aiken/dict.ak

```diff
@@ -148,11 +170,18 @@ test filter_3() {
 /// Finds a value in the dictionary, and returns the first key found to have that value.
 ///
 /// ```aiken
-/// dict.new()
-///   |> dict.insert(key: foo, value: 42, bytearray.compare)
-///   |> dict.insert(key: bar, value: 14, bytearray.compare)
-///   |> dict.find(42)
-/// // Some(foo)
+/// use aiken/bytearray
+///
+/// let foo: ByteArray = #"00"
+/// let bar: ByteArray = #"11"
+/// 
+/// let result =
+///   dict.new()
+///     |> dict.insert(key: foo, value: 42, compare: bytearray.compare)
+///     |> dict.insert(key: bar, value: 14, compare: bytearray.compare)
+///     |> dict.find(42)
```

Comment:
It'd be nice in this case to have another value holding 42, so we can illustrate how only the first / lowest key is returned.